src/Microsoft.Azure.ServiceBus/Resources.resx

```diff
@@ -216,4 +216,7 @@
   <data name="TimeoutMustBePositiveNonZero" xml:space="preserve">
     <value>Argument {0} must be a positive non-zero timeout value. The provided value was {1}.</value>
   </data>
+  <data name="MaxMessageCountMustBeGreaterThanZero1" xml:space="preserve">
+    <value>The specified value '{0}' is invalid. "maxMessageCount" must be greater than zero.</value>
```

Comment:
```suggestion
    <value>The specified value '{0}' is invalid. "MaxMessageCount" must be greater than zero.</value>
```
Also, I think the type including `MaxMessageCount` should be specified to ensure it's a clear exception message and users should know what to change.